wemake_python_styleguide/visitors/ast/subscripts.py

```diff
@@ -71,11 +71,19 @@ def _check_redundant_subscript(self, node: ast.Subscript) -> None:
             and not self._is_none(node.slice.step)
         )
 
-        if not (lower_ok and upper_ok and step_ok):
+        if not lower_ok:
             self.add_violation(
-                consistency.RedundantSubscriptViolation(
-                    node,
-                ),
+                consistency.RedundantSubscriptViolation(node.slice.lower)
+            )
+
+        if not upper_ok:
+            self.add_violation(
+                consistency.RedundantSubscriptViolation(node.slice.upper)
+            )
+
+        if not step_ok:
+            self.add_violation(
+                consistency.RedundantSubscriptViolation(node.slice.step)
```

Comment:
Can you please select a proper location once and once call `self.add_violation(consistency.RedundantSubscriptViolation(location))`?